webui/src/pages/auth/users/create-user-with-password.jsx

```diff
@@ -0,0 +1,112 @@
+import Layout from "../../../lib/components/layout";
+import React, {useRef, useState} from "react";
+import {Redirect, Route} from "react-router-dom";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import Card from "react-bootstrap/Card";
+import Form from "react-bootstrap/Form";
+import {auth} from "../../../lib/api";
+import {ActionGroup, ActionsBar, Error} from "../../../lib/components/controls";
+import Button from "react-bootstrap/Button";
+import {useRouter} from "../../../lib/hooks/router";
+import jwt_decode from "jwt-decode";
+
+const TOKEN_PARAM_NAME = "token";
+
+const CreateUserWithPasswordForm = ({token}) => {
+
+    const router = useRouter();
+    const decoded = jwt_decode(token);
+    const invitedUserEmail = decoded.sub;
+
+    const onConfirmPasswordChange = () => {
+        setPwdConfirmValid(true)
+        if (newPwdField.current !== null) {
+            const isPasswordMatch = newPwdField.current.value === confirmPasswordField.current.value;
+            setPwdConfirmValid(isPasswordMatch);
+            setFormValid(isPasswordMatch)
+        }
```

Comment:
Thanks, i'm not sure that I want to do `setPwdConfirmValid(true)` in case that there is an input in confirm but pwd is null. 
What do you think of displaying an error that says - "please insert you password first, and then confirm"? 
Alternatively I can add validation in case that password is filled after the confirm field.

Reply:
I just rewrote the above code using `else` to call `setPwdConfirmValid` just once in case `newPwdField.current === null`